azure-pipelines.yml

```diff
@@ -67,7 +67,7 @@ stages:
             -e NRN_RELEASE_UPLOAD \
             -e NEURON_WHEEL_VERSION \
             -e NRN_BUILD_FOR_UPLOAD=1 \
-            'neuronsimulator/neuron_wheel:latest-x86_64' \
+            'neuronsimulator/neuron_wheel:cpp17' \
```

Comment:
@alexsavulescu shall we just push the `cpp17` tag as the new `latest-x86_64`?

Reply:
That was just a build for testing purposes. We can do it just before the actual merge.

Reply:
> @alexsavulescu shall we just push the cpp17 tag as the new latest-x86_64?

👍